docs/cse/records-signals-entities-insights/signal-index-migration-faq.md

```diff
@@ -0,0 +1,257 @@
+---
+id: signal-index-migration-faq
+title: 2023 CSE Signal Index Migration FAQ
+sidebar_label: Signal Index Migration FAQ
+description: Learn about deprecation of the Signal Forwarding feature and what you need to know about migrating to the new sec_signal index.
+---
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+## Summary
+
+Starting last year, Signals generated by Cloud SIEM Enterprise (CSE) are automatically saved in a standardized `sec_signal` index. This special partition is similar to the existing `sec_record` indices in that, unlike data retained using the older Signal Forwarding feature, it is stored in a format that supports keyword search, nested attributes, and other standard log search features.
```

Comment:
I'd specify `2022` in case this doc lives on into 2024 and beyond

```suggestion
Effective in 2022, Signals generated by Cloud SIEM Enterprise (CSE) are automatically saved in a standardized `sec_signal` index. This special partition is similar to the existing `sec_record` indices in that, unlike data retained using the older Signal Forwarding feature, it is stored in a format that supports keyword search, nested attributes, and other standard log search features.
```